### ISSUE TITLE
Random traceback

### ISSUE THREAD

**Issue by scpketer** [Monika-After-Story/MonikaModDev#6181]
So I left the mod in the background, got to do my own business and stuff, and when I opened it after some time it greeted me with that.

```
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script-ch30.rpy", line 1427, in script call
  File "game/script-ch30.rpy", line 1596, in script
  File "game/script-ch30.rpy", line 1620, in <module>
  File "game/zz_windowreacts.rpy", line 217, in mas_checkForWindowReacts
  File "game/zz_windowreacts.rpy", line 191, in mas_isInActiveWindow
  File "game/zz_windowreacts.rpy", line 156, in mas_getActiveWindow
CalledProcessError: Command '[u'xdotool', u'getwindowfocus', u'getwindowname']' returned non-zero exit status 1

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script-ch30.rpy", line 1427, in script call
  File "game/script-ch30.rpy", line 1596, in script
  File "/DDLC-MAS/renpy/ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/DDLC-MAS/renpy/python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script-ch30.rpy", line 1620, in <module>
  File "game/zz_windowreacts.rpy", line 217, in mas_checkForWindowReacts
  File "game/zz_windowreacts.rpy", line 191, in mas_isInActiveWindow
  File "game/zz_windowreacts.rpy", line 156, in mas_getActiveWindow
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/subprocess.py", line 573, in check_output
CalledProcessError: Command '[u'xdotool', u'getwindowfocus', u'getwindowname']' returned non-zero exit status 1

Linux-5.7.12-arch1-1-x86_64-with-glibc2.2.5
Ren'Py 6.99.12.4.2187
Monika After Story 0.11.3
```

Seems pretty much like something related to window reactions?

**Comment by Booplicate** [comment]
Fixed in #6014, you'll have to disable window reactions 'til next release.

**Comment by scpketer** [comment]
Roger that.
